.github/workflows/changed_file_trigger_examples_check.yml

```diff
@@ -0,0 +1,117 @@
+name: change_examples_check_all
```

Comment:
Use a proper name for the workflow, this serves like a title
`Test Examples` will do.